scripts/test

```diff
@@ -3,13 +3,8 @@ set -e
 
 cd $(dirname "$BASH_SOURCE[0]")/..
 
-echo "running go fmt ..."
-res=$(go fmt ./...)
-if [ -n "$res" ]; then
-  echo "$res"
-  echo "gofmt failed..."
-  exit 1
-fi
+echo "running golangci-lint ..."
+golangci-lint run -E gofmt -E golint -E gocritic -E gosec -E misspell
```

Comment:
This should move into a config file so running `golangci-lint` locally is consistent with what is run in CI